src/ValueObject/ImageMediaValueObject.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme\ValueObject;
+
+use Drupal\image\Plugin\Field\FieldType\ImageItem;
+
+/**
+ * Handle information about a media item.
```

Comment:
Class description is not descriptive enough.

Reply:
done.